CHANGES.md

```diff
@@ -2,6 +2,11 @@
 
 ## __NEXT__
 
+### Features
+
+* filter: Add support to group by week (`--group-by week`) during subsampling. [#1067][] (@victorlin)
```

Comment:
I would detail that this is [ISO week](https://en.wikipedia.org/wiki/ISO_week_date) and not (one of the two definitions of) epiweek. Because we're grouping-by not filtering to specific weeks it's fine to use isoweek I think.

Reply:
@jameshadfield good point, I just read up on this briefly. Specifically the [epiweeks background page](https://epiweeks.readthedocs.io/en/stable/background.html) was helpful.

From my understanding, epidemiological weeks have two numbering systems:

1. US CDC (MMWR)
2. ISO

From the epiweeks page:

> The ISO defines the week ([ISO week](https://en.wikipedia.org/wiki/ISO_week_date)) as seven days, beginning with Monday and ending with Sunday

From [`date.isocalendar()` docs](https://docs.python.org/3/library/datetime.html#datetime.date.isocalendar):

> a week starts on a Monday and ends on a Sunday.

So this is actually implementing the 2nd definition, right?

Reply:
Oh yes, it looks like it! That's great, we can just say we're using the ISO Epiweek definition.

Reply:
Addressed in [force-push](https://github.com/nextstrain/augur/compare/89df17d3e7fad0c4e661e1b9c5287067b5c14908..c4df4607260a315eea6b3584e161c61a04a174cb).